canonical/juju-dashboard
# WD-5478 - Store fetched audit events in Redux

## Done

- Store audit events in Redux so that they can be accessed by the filter components as well as the table.

## QA

- Go to the logs page.
- Check that the audit logs are fetch and displayed in the table.

## Details

https://warthogs.atlassian.net/browse/WD-5478
